tests/validation/cattlevalidationtest/core/test_ldap.py

```diff
@@ -0,0 +1,181 @@
+from common_fixtures import *  # NOQA
+from requests.auth import AuthBase
+
+
+if_ldap = pytest.mark.skipif(not os.environ.get('API_AUTH_LDAP_SERVER'),
+                             reason='API_AUTH_LDAP_SERVER is not set')
+
+
+BASE_URL = cattle_url()
```

Comment:
No because the way the python clients make themselves they create functions based off of schema and token is only on the schema for token account. This is the same method used in github.